server/routes/routes_config.go

```diff
@@ -13,6 +13,15 @@ type AdminEndpoints interface {
 	RegisterAdminEndpoints(router *gin.RouterGroup)
 }
 
+// OldAPIEndpointsFunc wrapping type for function to mark it as implementation of OldApiEndpoints.
+type OldAPIEndpointsFunc func(router *gin.RouterGroup)
```

Comment:
We should use this one in all old handlers instead

Reply:
changes made